content/en/observability_pipelines/setup/datadog_with_archiving.md

```diff
@@ -0,0 +1,628 @@
+
+* Create a policy with the following permissions:
+
+  ```json
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Sid": "DatadogUploadLogArchives",
+        "Effect": "Allow",
+        "Action": ["s3:PutObject"],
+        "Resource": [
+          "arn:aws:s3:::<MY_BUCKET_NAME_1_/_MY_OPTIONAL_BUCKET_PATH_1>/*",
+        ]
+      },
+      {
+        "Sid": "DatadogListLogArchives",
+        "Effect": "Allow",
+        "Action": ["s3:ListBucket"],
+        "Resource": [
+          "arn:aws:s3:::<MY_BUCKET_NAME>"
+        ]
+      }
+    ]
+  }
+  ```
+
+* Edit the bucket name in the policy above to match your configuration.
+
+* Create an IAM user and attach the policy above to it. Create access credentials for the IAM user. Save these credentials as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+
+[1]: https://aws.amazon.com/s3/pricing/
+{{% /tab %}}
+{{% tab "Terraform (AWS)" %}}
+The sample Terraform configuration that you download later includes a sink for sending logs to Amazon S3 under a Datadog-rehydratable format. To use this configuration, set up an IAM policy that allows the Workers to write to S3.
+
+* Go into your AWS console and create an S3 bucket to send your archives to.
+
+  * Do not make your bucket publicly readable.
+  * For US1, US3, and US5 sites, see [AWS Pricing][1] for inter-region data transfer fees and how cloud storage costs may be impacted.
+
+* Create a policy with the following permissions:
+
+  ```json
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Sid": "DatadogUploadLogArchives",
+        "Effect": "Allow",
+        "Action": ["s3:PutObject"],
+        "Resource": [
+          "arn:aws:s3:::<MY_BUCKET_NAME_1_/_MY_OPTIONAL_BUCKET_PATH_1>/*",
+        ]
+      },
+      {
+        "Sid": "DatadogListLogArchives",
+        "Effect": "Allow",
+        "Action": ["s3:ListBucket"],
+        "Resource": [
+          "arn:aws:s3:::<MY_BUCKET_NAME>"
+        ]
+      }
+    ]
+  }
+  ```
+
+* Edit the bucket name in the policy above to match your configuration.
+
+* Attach the policy to the IAM Instance Profile that is created with Terraform, which you can find under the `iam-role-name` output.
+
+In order to run the Worker in your AWS account, you need administrative access to that account. Collect the following pieces of information to run the Worker instances:
+* The VPC ID your instances will run in.
+* The subnet IDs your instances will run in.
+* The AWS region your VPC is located in.
+
+[1]: https://aws.amazon.com/s3/pricing/
+{{% /tab %}}
+{{< /tabs >}}
+
+## Installing the Observability Pipelines Worker
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+The Observability Pipelines Worker Docker image is published to Docker Hub [here][1].
+
+1. Download the [sample pipeline configuration file][2].
+
+2. Run the following command to start the Observability Pipelines Worker with Docker:
+    ```
```

Comment:
Want to see if I can fix the markdown...for some reason the rest of the content is in blue

```suggestion

    ```shell
```
```